docs/generated/templates.md

```diff
@@ -160,6 +160,28 @@ KubeLinter supports the following templates:
   type: array
 ```
 
+## Dangling Service Monitor
+
+**Key**: `dangling-servicemonitor`
+
+**Description**: Flag service monitors which do not match any service
+
+**Supported Objects**: ServiceMonitor
+
+
+**Parameters**:
+
+```yaml
+- description: Option to disable checking if the namespace selector is dangling
```

Comment:
What is the purpose of these flags? Just wondering in what scenario one would only want to check one of the selectors but not the other. Isn't a ServiceMonitor dangling iff no service is found using the intersection of both selectors?

Reply:
I use kubelinter on helm charts that were broken up by namespace. Initially, I thought someone might have a serviceMonitor resource in a chart that points to a service defined in a separate namespace and thus helm chart.

If they were using only a namespaceSelector, then they might want to disable the namespaceSelector check... but I realised that even in that scenario, the check will still fail anyway. A more sensible solution for that usecase would be to annotate the resource to skip checking it.

@janisz can you think of any usecases?

Reply:
How about removing it. We can always add it but removal will be a breaking change.

Reply:
ok, have removed it.